addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3274,7 +3274,18 @@ const SnippetOptionWidget = Widget.extend({
 
         function applyCSS(cssProp, cssValue, styles) {
             if (!weUtils.areCssValuesEqual(styles[cssProp], cssValue, cssProp, this.$target[0])) {
-                this.$target[0].style.setProperty(cssProp, cssValue, 'important');
+                this.$target[0].style.setProperty(cssProp, cssValue);
+                // If change had no effect then make it important.
+                if (params.extraClass) {
+                    this.$target[0].classList.add(params.extraClass);
+                }
```

Comment:
I think the logic around this `extraClass` should either be refactored or explained via inline comments. I can't understand why we need to set it, remove it and then reset if the method returns true.

Reply:
Added a big comment to explain the `extraClass` shenanigan, and a comment before each of the two `if`s that requires the opposite situation.